logs/querysample/querysample.go

```diff
@@ -116,3 +133,19 @@ func TransformLogMinDurationStatementToQuerySample(logLine state.LogLine, queryT
 	}
 	return sample, true
 }
+
+func findQueryParameters(paramText string) []null.String {
+	// Handle Query Parameters (available from Postgres 16+)
+	var parameters []null.String
+	// Regular expression to find all values in single quotes or NULL
+	// Query Parameters example: $1 = 'foo', $2 = '123', $3 = NULL, $4 = 'bo''o'
```

Comment:
note: params are always quoted, regardless of the type (e.g. $2 is passed as integer but the output is quoted)

https://github.com/postgres/postgres/blob/4d93bbd4e0d414a33521f62d6249ac88486c866f/src/backend/nodes/params.c#L379